GymApp/GymAppApi/Extensions/ExceptionMiddlewareExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GYM.API.Models;
+using Microsoft.AspNetCore.Diagnostics;
+using System.Net;
+
+namespace GYM.API.Extensions
+{
+    public static class ExceptionMiddlewareExtensions
+    {
+        public static void ConfigureExceptionHandler(this IApplicationBuilder app, ILogger logger)
+        {
+            app.UseExceptionHandler(appError =>
```

Comment:
Add custom exeption handler with using try catch 
Read abount method in middlewares